src/handlers.py

```diff
@@ -14,25 +21,37 @@ class UserInput(StatesGroup):
     tg_id = State()
 
 
+class SendingMail(StatesGroup):
+    email = State()
+
+
 instance = CallbackData("button", "action")
 
 
-def get_keyboard():
+def get_keyboard(user_tg_id):
     keyboard = types.InlineKeyboardMarkup(row_width=1)
-    buttons = [
-        types.InlineKeyboardButton(text="Зарегистрироваться", callback_data=instance.new(action="registrate")),
-        types.InlineKeyboardButton(text="Доп. информация", callback_data=instance.new(action="info")),
-        types.InlineKeyboardButton(text="Отменить регистрацию", callback_data=instance.new(action="cancel")),
-    ]
-    keyboard.add(*buttons)
+    driver = DBDriver()
+    user_tg_id_from_db = driver.get_user(user_tg_id)
+    if user_tg_id == user_tg_id_from_db:
```

Comment:
This expression will become much simpler if you rewrite and rename _get_user()_ method